src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetRandomCommand.cs

```diff
@@ -28,6 +28,7 @@ public class GetRandomCommand : PSCmdlet
 
         private const string RandomNumberParameterSet = "RandomNumberParameterSet";
         private const string RandomListItemParameterSet = "RandomListItemParameterSet";
+        private const string ShuffleSet = "ShuffleSet";
```

Comment:
```suggestion
        private const string ShuffleSet = "ShuffleParameterSet";
```

Just to keep it in the same vein as the existing parameter set names, really. 🙂